apps/okreads-e2e/src/integration/reading-list.spec.ts

```diff
@@ -11,4 +13,26 @@ describe('When: I use the reading list feature', () => {
       'My Reading List'
     );
   });
+
+  it('Then: I should be able to mark the book as finished', () => {
+    cy.get('input[type="search"]').type('angular');
+
+    cy.get('form').submit();
+
+    cy.get('[data-testing="add-to-reading-list"]:enabled').first().click();
+
+    cy.get('[data-testing="toggle-reading-list"]').click();
+
+    cy.get('[data-testing="reading-list-item"]').should('have.lengthOf.above', readingItemsCount);
```

Comment:
Don't understand this assertion when you can test for exact count. Also your assertions should match whatever is described in the 'it' block. (line 17)